src/App.js

```diff
@@ -1,30 +1,10 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { PureComponent } from 'react';
 import './App.css';
+import Calculator from './Components/calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends PureComponent {
```

Comment:
- [ ] Your app doesn't run properly, you can check your code to see if everything is installed correctly, I got this error when trying to open the app for preview. kindly consider fixing it

![Screenshot_20220810_175447](https://user-images.githubusercontent.com/25227022/183971483-9a5e3962-851a-4d37-b64a-3f722cc9018e.png)